Expensify/App
# Add server logs to help diagnose report data not updating

### Details
I cannot reproduce the bug people are experiencing [here](https://github.com/Expensify/Expensify/issues/153315) so I am added some server logs to hopefully get more information when this happens to someone else. Also adding some logs for when the user goes active or background so that it's easier to pinpoint at what point they came back and whether any reconnection callbacks fired.

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/153315

### Tests
1. Load up app and tail server logs
2. Move app to background
3. See that there is a server log for that event
4. Wait 10 seconds or so
4. Move app to foreground
5. See that there is another server log saying you became active and also one to fire reconnection callbacks
6. Log out and log back in and verify there are no issues doing so

```
2021-02-16T02:44:04.823047+00:00 expensidev php-fpm: mQ4xBG /api.php marc@expensify.com !https://expensify.cash/! ?api? [info] [info] [NetworkConnection] listenForReconnect called ~~ parameter: '[]' userAgent: 'Expensify.cash/438 CFNetwork/1206 Darwin/19.6.0'
2021-02-16T02:44:16.805952+00:00 expensidev php-fpm: blTF97 /api.php marc@expensify.com !https://expensify.cash/! ?api? [info] [info] [NetworkConnection] AppState change event ~~ parameter: '{"state":"inactive"}' userAgent: 'Expensify.cash/438 CFNetwork/1206 Darwin/19.6.0'
2021-02-16T02:44:17.336935+00:00 expensidev php-fpm: G2xFbY /api.php marc@expensify.com !https://expensify.cash/! ?api? [info] [info] [NetworkConnection] AppState change event ~~ parameter: '{"state":"background"}' userAgent: 'Expensify.cash/438 CFNetwork/1206 Darwin/19.6.0'
2021-02-16T02:44:33.990856+00:00 expensidev php-fpm: mHpPx1 /api.php marc@expensify.com !https://expensify.cash/! ?api? [info] [info] [NetworkConnection] Firing reconnection callbacks because sleep timer clock skewed ~~ parameter: '[]' userAgent: 'Expensify.cash/438 CFNetwork/1206 Darwin/19.6.0'
2021-02-16T02:44:34.238455+00:00 expensidev php-fpm: Jc7Zcr /api.php marc@expensify.com !https://expensify.cash/! ?api? [info] [info] [NetworkConnection] AppState change event ~~ parameter: '{"state":"active"}' userAgent: 'Expensify.cash/438 CFNetwork/1206 Darwin/19.6.0'
```
### Tested On


### Screenshots


#### Web


#### Mobile Web


#### Desktop


#### iOS


#### Android
